### ISSUE TITLE
How to not show all the menu and sidebar

### ISSUE THREAD

**Issue by hhlim333** [man-group/dtale#796]
Hi,

Can I delete all the menu and side bar when showing the dataframe?

Just show the dataframe and not display ribbon-menu.

Thank you

**Comment by aschonfeld** [comment]
I can certainly add a configuration parameter to hide the ribbon menu.  Just curious what you mean by "side bar"?  Happy to add a parameter to hide that as well

**Comment by hhlim333** [comment]
Thank you for your quick response.

<img width="1075" alt="螢幕截圖 2023-09-08 13 28 17" src="https://github.com/man-group/dtale/assets/118234430/f63836d1-7503-4ccf-8b04-a46ee2860822">
<img width="1045" alt="螢幕截圖 2023-09-08 13 28 21" src="https://github.com/man-group/dtale/assets/118234430/f157e6b2-0b74-4f6f-8d86-0200fb52d23c">

Thank you for your help

**Comment by aschonfeld** [comment]
Ok, so you want the main menu and column menus hidden as well.  I can add flags for those as well

**Comment by hhlim333** [comment]
Am I correct ?

It seems not working.

<img width="1241" alt="螢幕截圖 2023-09-10 15 27 43" src="https://github.com/man-group/dtale/assets/118234430/193a63de-1184-49c8-a8ae-6798ea63dd7a">

Thank you for your help

**Comment by aschonfeld** [comment]
@hhlim333  so I just published v3.5.0 which includes this functionality.  You can hide the ribbon & side menus by specifying `hide_header_menu`, `hide_main_menu` and/or `hide_column_menus` when calling `dtale.show` (would only affect this piece of data):

```
import numpy as np
import pandas as pd
import dtale

dtale.show(pd.DataFrame([1,2,3,4,5]), hide_header_menu=True, hide_main_menu=True, hide_column_menus=True)
```

Another would be using global configuration. Following this [documentation](https://github.com/man-group/dtale/blob/master/docs/CONFIGURATION.md) you can add the following to the [app] section for it to be applied to every dataframe you load into D-Tale without specifying parameters in your code.  This would be applied to every piece of data you load into D-Tale:

```
hide_header_menu = True
hide_main_menu = True
hide_column_menus = True
```

And lastly you can lock the header menu for all data loaded into D-Tale thru code using the following:
```
import numpy as np
import pandas as pd
import dtale
import dtale.global_state as global_state

global_state.set_app_settings(dict(hide_header_menu=True, hide_main_menu=True, hide_column_menus=True))

dtale.show(pd.DataFrame([1,2,3,4,5]))
```

Let me know if you have any issues.
